tests/clients/test_experiment.py

```diff
@@ -216,6 +229,11 @@ def test_experiment_run_tag_schema():
     assert data == TAG
 
 
+def test_experiment_run_tag_schema_dump():
```

Comment:
This should probably just be `test_tag_schema_dump`.

Usually test names are in the format `test_foo_bar_method_name` when testing `FooBar.method_name`. If other tests break this we should fix. (Exceptions are usually made for schemas only used to convert in one direction, as is the case in most client modules in this library).

Reply:
OK, I renamed it, along with other similar tests.